libs/output-mapping/src/Writer/FileWriter.php

```diff
@@ -114,7 +114,9 @@ public function uploadFiles(
             }
             try {
                 $storageConfig = TagsHelper::addSystemTags($storageConfig, $systemMetadata, $this->logger);
-                $storageConfig = TagsHelper::rewriteTags($storageConfig, $this->clientWrapper);
+                if (!$this->clientWrapper->getClientOptionsReadOnly()->useBranchStorage()) {
```

Comment:
Akorát se vypiná prefixovaní, to, že se to uloží do branch storage zařidí magie v getTableAndFileStorageClient https://github.com/keboola/storage-api-php-client-branch-wrapper/blob/1fc72b82cb0661836f5733d81af5013a2e23829c/src/ClientWrapper.php#L61
řidí se to stejným flagem (useBrancStorage)